sec_certs/dataset/common_criteria.py

```diff
@@ -179,11 +179,10 @@ def from_web_latest(cls):
             return cls.from_json(dset_path)
 
     @property
-    def get_all_cert_id_references(self):
+    def all_cert_ids(self):
```

Comment:
Can you check if the following code is equivalent to yours?

```python
@property
def all_cert_ids(self):
    all_cert_ids = set([x.heuristics.cert_id for x in self])
    all_cert_ids.update(itertools.chain.from_iterable([x.pdf_data.keywords_rules_cert_id for x in self]))
    all_cert_ids.update(
        itertools.chain.from_iterable(
            x.pdf_data.st_keywords["rules_cert_id"] for x in self if x.pdf_data.st_keywords
        )
    )
    all_cert_ids.discard(None)
    return all_cert_ids
```

Decide which one you like more, just suggesting a shorter variant (but yours probably runs faster). We have `itertools.chain.from_iterable()` on a bambillion of places in the project anyway...